site/_includes/footer.html

```diff
@@ -14,6 +14,7 @@ <h3>Pages</h3>
 			<a href="/faq" rel="noopener">FAQ</a>
 			<a href="/feedback" rel="noopener">Send Feedback</a>
 			<a href="/contributors" rel="noopener">Contributors</a>
+					<a href="/privacy">Privacy Policy</a>
```

Comment:
We should add a rel="noopener" attribute here

Reply:
Added!